http-server/src/test/java/io/airlift/http/server/TestHttpServerConfig.java

```diff
@@ -121,6 +122,7 @@ public void testExplicitPropertyMappings()
                 .put("http-server.admin.threads.min", "3")
                 .put("http-server.admin.threads.max", "4")
                 .put("http-server.max-request-header-size", "32kB")
+                .put("http-server.max-response-header-size", "32kB")
```

Comment:
Can you make this `57k`?  That is different from the value above and unlikely to ever be a default.

Reply:
okay, I have modified